# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/base/PresShell.cpp
# Commit: 936591fa4a69
# Full Hash: 936591fa4a69557098f1ac1892211f78c005c9c6
# Author: Edgar Chen <echen@mozilla.com>
# Date: 2021-01-05 21:56:58
# Regressor Bug: 1653949
# File Overlap Count: 1
# Description:
#   Bug 1662363 - Rename eTopLevel/eChild to ePlatformTopLevel/ePlatformChild; r=smaug
#   
#   This is a follow-up of bug 1653949.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D100729
# ==============================================================================

diff -r ff581d36e640 -r 936591fa4a69 layout/base/PresShell.cpp
--- a/layout/base/PresShell.cpp	Tue Jan 05 09:47:53 2021 +0000
+++ b/layout/base/PresShell.cpp	Tue Jan 05 09:48:24 2021 +0000
@@ -7012,7 +7012,7 @@
   bool isWindowLevelMouseExit =
       (aGUIEvent->mMessage == eMouseExitFromWidget) &&
       (mouseEvent &&
-       (mouseEvent->mExitFrom.value() == WidgetMouseEvent::eTopLevel ||
+       (mouseEvent->mExitFrom.value() == WidgetMouseEvent::ePlatformTopLevel ||
         mouseEvent->mExitFrom.value() == WidgetMouseEvent::ePuppet));
 
   // Get the frame at the event point. However, don't do this if we're